src/mantine-core/src/Menu/MenuTarget/MenuTarget.tsx

```diff
@@ -35,12 +35,7 @@ export const MenuTarget = forwardRef<HTMLElement, MenuTargetProps>(
       () => ctx.trigger === 'hover' && ctx.closeDropdown()
     );
 
-    useEffect(
-      () => () => {
-        ctx.closeDropdown();
-      },
-      [children]
-    );
+    useEffect(() => ctx.closeDropdown(), [children]);
```

Comment:
nope, should be `useEffect(() => ctx.closeDropdown, [children])`

Reply:
my bad